kube-derive/src/custom_resource.rs

```diff
@@ -599,6 +599,6 @@ mod tests {
             struct FooSpec { foo: String }
         };
         let input = syn::parse2(input).unwrap();
-        let kube_attrs = KubeAttrs::from_derive_input(&input).unwrap();
+        let _kube_attrs = KubeAttrs::from_derive_input(&input).unwrap();
```

Comment:
i think this is fixed in master

Reply:
Doesn't look like it, the only PR that wasn't included here was #931, which didn't touch this.

Reply:
oh, my bad. i have a different fix for it (by testing more) in https://github.com/kube-rs/kube-rs/pull/924/files which isn't reviewed yet

Reply:
Ahh